### PR TITLE
refactor/common_qa_speak

### PR DESCRIPTION
**Pull request by JarbasAl**
move the self.speak call to the skill class where it belongs

companion PR https://github.com/OpenVoiceOS/OVOS-workshop/pull/63

### CODE REVIEW

File: mycroft/skills/intent_services/commonqa_service.py [hunk 1]
```diff
@@ -194,7 +195,11 @@ def _query_timeout(self, message):
                     pass
 
                 # invoke best match
-                self.speak(best['answer'])
+                # TODO verify version before PR merge
+                from ovos_workshop.version import VERSION_MAJOR, VERSION_ALPHA, VERSION_BUILD, VERSION_MINOR
```
Comment (NeonDaniel) [inline mycroft/skills/intent_services/commonqa_service.py:199]: could workshop add something to the message data instead of an installed version check?

Comment (JarbasAl) [reply mycroft/skills/intent_services/commonqa_service.py:199]: not sure... i dont want this to be supported via bus message as a public api, nothing should build on that, in the end it is messagebus spam as it is not supposed to be optional, we can detect its presence in bus or check the version, but things are kinda exclusive

will make the data come in message as a __private key but the reasoning is for skills runnings in their own venv and such, will keep the version check to log warnings and fully deprecate it in 0.1.0

Comment (NeonDaniel) [reply mycroft/skills/intent_services/commonqa_service.py:199]: If version checks are used here, they should include the alpha version as well to ensure alpha versions act as expected

Comment (JarbasAl) [reply mycroft/skills/intent_services/commonqa_service.py:199]: i left a TODO comment so we could sync the merge of two PRs and add that in, if any repo gets a PR merged meanwhile this needs updating

**Pull request merged**